raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -19,7 +19,7 @@ contract MonitoringService is Utils {
 
     // keccak256(channel_identifier, token_network_address) => Struct
     // Keep track of the rewards per channel
-    mapping(bytes32 => Reward) rewards;
+    mapping(bytes32 => Reward) public rewards;
```

Comment:
Is there a use case you have in mind for accessing this? I'd rather have it private otherwise.

Reply:
No, I had nothing in particular.  I'll say `private` here.